fe-nc-news-react/api.js

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+const api = axios.create({
+  baseURL: "https://be-nc-newsapi.onrender.com",
+  timeout: 5000,
+});
```

Comment:
Your articles aren't loading for me due to the timeout. Please remove it as it isn't needed